docker/setup-hosts.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+NODES=3
+
+
+## Create multiple hosts
+for N in $(seq 1 $NODES)
+do
+docker-machine create --driver virtualbox node$N
+done
+
+
+## Set up swarm
+### node1 as manager and others as workers
+SWARM_ADDR=$(docker-machine ip node1)
+
+docker-machine ssh node1 docker swarm init --advertise-addr $SWARM_ADDR
+
+SWARM_TOKEN=$(docker-machine ssh node1 docker swarm join-token -q worker)
+
+for N in $(seq 2 $NODES)
+do
+docker-machine ssh node$N docker swarm join --token $SWARM_TOKEN $SWARM_ADDR:2377
```

Comment:
Ok, I tried that with latest release 1.12.6 and no joy either, so I will leave it as is.